src/components/Feed/Feed.js

```diff
@@ -10,19 +9,16 @@ import AppContext from '../../Context/AppContext';
 
 const Feed = ({ profileIsVisible, searchResults }) => {
   const [currentUserPosts, setCurrentUserPosts] = useState([])
-  const allPosts = useContext(AppContext)
+  const allData = useContext(AppContext)
```

Comment:
I'm glad we walked through this together to better understand what we were receiving here